package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "quorum-dev-quickstart",
-  "version": "0.0.47",
+  "version": "0.0.48",
```

Comment:
@antonydenyer also needs this one here https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/npm-shrinkwrap.json#L3

Reply:
Give me a yell when this is good and I'll merge both and cut a release for it and ping you